.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -140,7 +140,32 @@ steps:
       bash hack/scripts/validate_state.sh
     name: "validatePods"
     displayName: "Validate Pods"
-  
+
+  - script: |
```

Comment:
Can we add this a script to the `hack` folder so that we don't have to change the script at multiple places. And call that script here.

Reply:
+1 but put it in `build/scripts/` not `hack/`

Reply:
instead of build, imo it should be placed in test/network.. i will create one